VIS/Areas/VIS/Controllers/PaymentAllocationController.cs

```diff
@@ -201,6 +224,24 @@ public JsonResult GetDocType()
             return Json(JsonConvert.SerializeObject(payments.GetDocType()));
         }
 
+        /// <summary>
+        /// to get DataTypes
+        /// </summary>
+        /// <returns>List of Data Types</returns>
+        public JsonResult GetpayDocType()
+        {
+            Ctx ct = Session["ctx"] as Ctx;
+            PaymentAllocation payments = new PaymentAllocation(ct);
+            return Json(JsonConvert.SerializeObject(payments.GetpayDocType()));
+        }
+
+        public JsonResult GetPaymentType()
```

Comment:
do commenting

Reply:
Okay sir!

Reply:
Payment Allocation, done the Changes in GL Grid.